src/ILLink.CodeFix/ILLink.CodeFixProvider.csproj

```diff
@@ -2,15 +2,33 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <IsPackable>false</IsPackable>
```

Comment:
I think we should set `AutoGenerateAssemblyVersion` to follow [this](https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/Documentation/CorePackages/Versioning.md#recommended-settings) arcade recommendation.